app/javascript/services/analysis-cached.js

```diff
@@ -699,7 +742,7 @@ export const fetchGladAlerts = ({
     adm1,
     adm2,
     grouped,
-    glad: true
+    allowedParams: 'glad'
```

Comment:
Better. Thank you.

Reply:
> You seem to have got it all working! Nothing wrong with any of the analysis building. I have the dynamic sentences not working on staging but maybe this isnt ready yet?

Not yet, no - need some new helper functions that depend on the min/max dates of the brush.